inbm/dispatcher-agent/tests/unit/source/test_source_command.py

```diff
@@ -226,7 +226,7 @@ def test_do_source_command_add(
             "dispatcher.source.source_command.create_application_source_manager",
             OsType.Ubuntu,
             ApplicationUpdateSourceParameters(
-                file_name="filename", sources=["source1", "source2"]
+                source_list_file_name="filename", sources=["source1", "source2"]
```

Comment:
**Maintainability**
**Suggestion:** The parameter name source_list_file_name should be consistent across all instances. If the change to source_list_file_name is intentional and the class ApplicationUpdateSourceParameters has been updated accordingly, ignore this suggestion. Otherwise, revert to the original parameter name file_name for consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            ApplicationUpdateSourceParameters(
                file_name="filename", sources=["source1", "source2"]
            ),
```